cookbook/assetic/asset_management.rst

```diff
@@ -204,7 +204,7 @@ Combining Assets
 ~~~~~~~~~~~~~~~~
 
 One feature of Assetic is that it will combine many files into one. This helps
-to reduce the number of HTTP requests, which is great for front end performance.
+to reduce the number of HTTP requests, which is great for frontend performance.
```

Comment:
both Bower and Bootstrap (on their homepages) use `front-end`. That looks a bit odd to me, but I guess we should follow their lead.